app/wrappers/prospect_api_wrapper.rb

```diff
@@ -0,0 +1,49 @@
+class ProspectApiWrapper
+  require 'rest-client'
+  include Singleton
+
+  API_KEY = ENV['PROSPECTS_API_KEY'].freeze
```

Comment:
This is rather uncommon :) Consider just calling `ENV.fetch("PROSPECTS_API_KEY")` where necessary